MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -888,6 +1040,57 @@ void EnggDiffractionViewQtGUI::browseTextureDetGroupingFile() {
   m_uiTabFocus.lineEdit_texture_grouping_file->setText(path);
 }
 
+void EnggDiffractionViewQtGUI::browseFitFocusedRun() {
+  QString prevPath = QString::fromStdString(m_focusDir);
+  if (prevPath.isEmpty()) {
+    prevPath =
+        MantidQt::API::AlgorithmInputHistory::Instance().getPreviousDirectory();
+  }
+  std::string nexusFormat = "Nexus file with calibration table: NXS, NEXUS"
+                            "(*.nxs *.nexus);;";
+
+  QString path(
+      QFileDialog::getOpenFileName(this, tr("Open Focused File "), prevPath,
+                                   QString::fromStdString(nexusFormat)));
+
+  if (path.isEmpty()) {
+    return;
+  }
+
+  MantidQt::API::AlgorithmInputHistory::Instance().setPreviousDirectory(path);
+  m_uiTabFitting.lineEdit_pushButton_run_num->setText(path);
+}
+
+void EnggDiffractionViewQtGUI::browsePeaksToFit() {
+
+  try {
+    QString prevPath = QString::fromStdString(m_focusDir);
+    if (prevPath.isEmpty()) {
+      prevPath = MantidQt::API::AlgorithmInputHistory::Instance()
+                     .getPreviousDirectory();
+    }
+
+    QString path(
+        QFileDialog::getOpenFileName(this, tr("Open Peaks To Fit"), prevPath,
+                                     QString::fromStdString(g_DetGrpExtStr)));
```

Comment:
Ah, this string can be confusing in this context. We should check with scientists the names/hints that they'd prefer to use for these files in this tab.